src/sentry/eventstream/snuba.py

```diff
@@ -469,6 +470,7 @@ def insert(
             **kwargs,
         )
         self._dispatch_post_process_group_task(
+            EventStream._get_event_type(event),
```

Comment:
If you need to call this method outside of the class it should probably be a public method.